examples/postgres/README.md

```diff
@@ -0,0 +1,117 @@
+# Postgres INSERT/SELECT Function Image
+
+This function executes an INSERT or SELECT against a table in a given postgres server.
+
+## Requirements
+
+- Postgres Server
+- IronFunctions API
+
+## Development
+
+### Building image locally
+
+```
+# SET BELOW TO YOUR DOCKER HUB USERNAME
+USERNAME=YOUR_DOCKER_HUB_USERNAME
+
+# build it
```

Comment:
I know some of our examples need updating to the cli commands, but could you change the README to be like the `helloworld` examples that use the cli (much simpler).  Like this: https://github.com/iron-io/functions/tree/master/examples/hello/go

Even if just for the build, push, bump so you can get rid of the `.sh` files and simplify things a bit.